### PR TITLE
[TASK] Raise PHP Stan level to 8

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #217

### CODE REVIEW

File: Classes/Domain/Model/Product/Tea.php
Comment (oliverklee) [inline Classes/Domain/Model/Product/Tea.php]: The code sniffer needs this to be on the line above.

But apart from that, I'd prefer to have the issues in a baseline file (like [in this project](https://github.com/sabberworm/PHP-CSS-Parser)) so we have a list of things we probably need to fix.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Comment by flicstar** [comment]
@oliverklee PING. It is our Sweeping Week when we chase up open PRs. Looks like this one is waiting on a review from you. Just wanted to make sure it's in your queue. Thanks!

**Pull request closed without merging**
